<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-9P1G2QK" failures="2" tests="4" name="Test And ReportNG" time="15.696" errors="0" timestamp="2023-12-20T13:38:09 BDT">
  <testcase classname="generatingReports_And_reportNG.Assertion_Test_Employee_Login" name="homePageVerification" time="2.673"/>
  <testcase classname="generatingReports_And_reportNG.Assertion_Test_Employee_Login" name="test_Create_Order" time="4.09"/>
  <testcase classname="generatingReports_And_reportNG.Assertion_Test_Employee_Login" name="test_Customer_Login" time="6.649">
    <failure type="org.opentest4j.AssertionFailedError" message="Locator expected to have text: Welcome Davidd
Received: Welcome David

Call log:
Locator.expect with timeout 5000ms
waiting for locator(&amp;quot;xpath=//*[@id=&amp;apos;divimg&amp;apos;]/div/h2[2]&amp;quot;)
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;
  locator resolved to &amp;lt;h2&amp;gt;Welcome David&amp;lt;/h2&amp;gt;
  unexpected value &amp;quot;Welcome David&amp;quot;">
      <![CDATA[org.opentest4j.AssertionFailedError: Locator expected to have text: Welcome Davidd
Received: Welcome David

Call log:
Locator.expect with timeout 5000ms
waiting for locator("xpath=//*[@id='divimg']/div/h2[2]")
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
  locator resolved to <h2>Welcome David</h2>
  unexpected value "Welcome David"
at com.microsoft.playwright.impl.AssertionsBase.expectImpl(AssertionsBase.java:73)
at com.microsoft.playwright.impl.AssertionsBase.expectImpl(AssertionsBase.java:50)
at com.microsoft.playwright.impl.AssertionsBase.expectImpl(AssertionsBase.java:41)
at com.microsoft.playwright.impl.LocatorAssertionsImpl.hasText(LocatorAssertionsImpl.java:214)
at com.microsoft.playwright.assertions.LocatorAssertions.hasText(LocatorAssertions.java:1442)
at base.Base.assertion(Base.java:79)
at generatingReports_And_reportNG.Assertion_Test_Employee_Login.test_Customer_Login(Assertion_Test_Employee_Login.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test_Customer_Login -->
  <testcase classname="generatingReports_And_reportNG.Assertion_Test_Employee_Login" name="test_Employee_Login" time="1.695">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Employee Id: 102] but found [Employee Id: 1022]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Employee Id: 102] but found [Employee Id: 1022]
at generatingReports_And_reportNG.Assertion_Test_Employee_Login.test_Employee_Login(Assertion_Test_Employee_Login.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_Employee_Login -->
</testsuite> <!-- Test And ReportNG -->
